### ISSUE TITLE
Seedvaut Crash when attempting backup via WebDAV mount

### ISSUE THREAD

**Issue by KingE2** [stevesoltys/backup#378]
https://user-images.githubusercontent.com/98594394/151552091-dfd2f1f9-ba90-4fcb-8b00-6184a8ba565f.mp4

1.  Installed and started WebDAV server on my synology NAS.
2. Successfully connected to the WebDav server from Pixel 6 running Android 12 with Calyx 3.1.1.  Verified that I can browse the Synology Disk from Pixel.
3. Opened SeedVault, attempted to initate a backup to WebDAV.  Seedvault sometimes shows an apparent "close then immediate re-open" of the application. // Other times it shows a "initializing Backup Location" and then displays a crash dialoge "

**Comment by grote** [comment]
Could you please also provide a log via logcat from that crash?

**Comment by KingE2** [comment]
Is this helpful?  I pressed record, then tried to run a backup a few times.

<<< log_count = 10 >>>
[01-28 12:16:54.883 20052:20070 D/OpenGLRenderer]
endAllActiveAnimators on 0xb40000718d325300 (MenuPopupWindow$MenuDropDownListView) with handle 0xb40000709d1f5220

[01-28 12:17:00.203 20052:20052 D/LogcatReader]
[Logcat] onActivityInBackground

[01-28 12:17:00.607 20052:20070 D/OpenGLRenderer]
endAllActiveAnimators on 0xb40000718d23ea50 (RippleDrawable) with handle 0xb40000709d1ffd50

[01-28 12:17:01.811 20052:20052 D/LogcatReader]
[Logcat] onActivityInForeground

[01-28 12:17:01.811 20052:20052 D/LogcatReader]
[Logcat] Posting pending logs

[01-28 12:17:01.833 20052:20052 D/LogcatReader]
[LogcatLiveFragment] onServiceConnected

[01-28 12:17:03.735 20052:20052 D/LogcatReader]
[Logcat] onActivityInBackground

[01-28 12:17:21.208 20052:20052 D/LogcatReader]
[Logcat] onActivityInForeground

[01-28 12:17:21.208 20052:20052 D/LogcatReader]
[Logcat] Posting pending logs

[01-28 12:17:21.227 20052:20052 D/LogcatReader]
[LogcatLiveFragment] onServiceConnected

**Comment by NDLcode** [comment]
I had the exact same issue with Synology NAS WebDAV server. Now it is working great. I did the following : 

Initially I created a DAV mount using address : http://my_nas:5005 then seedvault kept crashing. However I could see my mount working well form DAVx5 app.

- Deleted the mount from DAVx5 app.
- Created dedicated CalyxOS user on my NAS with WebDAV rights.
- Created dedicated **shared folder** for calyx backup on NAS and gave R/W access to above created user.
- Recreated DAV mount using DAV URL as : http://my_nas:5005/backup_shared_folder and logged with NAS dedicated user.
- restarted the phone

It now works like a charm and it is amazingly fast compared to previous nextcloud pain.

Hope this helps
